compiler/compiler.go

```diff
@@ -70,12 +70,12 @@ func polyType(t semantic.PolyType, err error) semantic.PolyType {
 }
 
 // compile recursively compiles semantic nodes into evaluators.
-func compile(n semantic.Node, typeSol semantic.TypeSolution, scope Scope, funcExprs map[string]*semantic.FunctionExpression) (Evaluator, error) {
+func compile(scope Scope, funcExprs map[string]*semantic.FunctionExpression, n semantic.Node, typeSol semantic.TypeSolution) (Evaluator, error) {
```

Comment:
Should these changes just be reverted since they just change the order of parameters?